apps/skilavottord/web/screens/DeregisterVehicleKm/Confirm/ConfirmKM.tsx

```diff
@@ -0,0 +1,288 @@
+import { useMutation, useQuery } from '@apollo/client'
+import gql from 'graphql-tag'
+import { useRouter } from 'next/router'
+import React, { FC, useContext, useEffect } from 'react'
+
+import {
+  Box,
+  Bullet,
+  BulletList,
+  Button,
+  Hidden,
+  Inline,
+  LoadingDots,
+  Stack,
+  Text,
+  toast,
+} from '@island.is/island-ui/core'
+
+import { hasPermission } from '@island.is/skilavottord-web/auth/utils'
+import {
+  CarDetailsBox,
+  NotFound,
+  OutlinedError,
+  ProcessPageLayout,
+} from '@island.is/skilavottord-web/components'
+import { UserContext } from '@island.is/skilavottord-web/context'
+import {
+  Mutation,
+  Query,
+  RecyclingRequestTypes,
+  RequestErrors,
+  RequestStatus,
+  Role,
+} from '@island.is/skilavottord-web/graphql/schema'
+import { useI18n } from '@island.is/skilavottord-web/i18n'
+import { getYear } from '@island.is/skilavottord-web/utils/dateUtils'
+import { useForm } from 'react-hook-form'
+
+const SkilavottordVehicleReadyToDeregisteredQuery = gql`
+  query skilavottordVehicleReadyToDeregisteredQuery($permno: String!) {
+    skilavottordVehicleReadyToDeregistered(permno: $permno) {
+      vehicleId
+      vehicleType
+      newregDate
+      recyclingRequests {
+        nameOfRequestor
+      }
+      mileage
+    }
+  }
+`
+
+const SkilavottordRecyclingRequestMutation = gql`
+  mutation skilavottordRecyclingRequestMutation(
+    $permno: String!
+    $requestType: RecyclingRequestTypes!
+  ) {
+    createSkilavottordRecyclingRequest(
+      permno: $permno
+      requestType: $requestType
+    ) {
+      ... on RequestErrors {
+        message
+        operation
+      }
+      ... on RequestStatus {
+        status
+      }
+    }
+  }
+`
+
+const UpdateSkilavottordVehicleMileageMutation = gql`
+  mutation updateSkilavottordVehicleMileage(
+    $permno: String!
+    $mileage: Float!
+  ) {
+    updateSkilavottordVehicleMileage(permno: $permno, mileage: $mileage)
+  }
+`
+
+const Confirm: FC<React.PropsWithChildren<unknown>> = () => {
+  const {
+    control,
+    watch,
+    formState: { errors },
+  } = useForm({
+    mode: 'onChange',
+  })
```

Comment:
`formState: { errors },` is unused, can be removed
```suggestion
  const { control, watch } = useForm({
    mode: 'onChange',
  })
```

Reply:
Fixed